requirements.txt

```diff
@@ -4,28 +4,28 @@ cryptography==38.0.1
 decorator==5.1.1
 eventlet==0.30.2
 Flask-Login==0.6.2
-Flask-SocketIO==5.3.1
-Flask-SQLAlchemy==3.0.2
+Flask-SocketIO==5.3.2
+Flask-SQLAlchemy==3.0.3
 Flask==2.2.2
 gunicorn==20.1.0
 lti==0.9.5
 oauthlib==3.1.1
 psycopg2-binary==2.9.5
 python-dateutil==2.8.2
 python-magic==0.4.27
-pytz==2022.6
-requests==2.28.1
-simplejson==3.17.6
-smart-open==4.2.0
-SQLAlchemy==1.4.42
-sqlvalidator==0.0.19
-Werkzeug==2.2.2
+pytz==2022.7.1
+requests==2.28.2
+simplejson==3.18.3
+smart-open==4.1.2
```

Comment:
Travis is green after I downgraded smart-open from 4.2.0 to 4.1.2.  I'll create a Jira to upgrade to smart-open v5.  Not sure when v4.2.0 will fix itself.